### PR TITLE
RPM update

### PR DESCRIPTION
**Pull request by calccrypto**
man pages used to be placed into `gufi.rpm` no matter what. Now they are placed into `gufi-server.rpm` and `gufi-client.rpm`.
Fixed bug with installing man pages.
rpms depend on `python2 >= 2.7` not `python >= 2.7, python < 3` to make them work on CentOS 8.

### CODE REVIEW

**Comment by calccrypto** [comment]
Currently getting

```
Transaction check error:
  file /usr/share/man from install of GUFI-0.5.1-1.x86_64 conflicts with file from package filesystem-3.2-25.el7.x86_64
  file /usr/share/man/man1 from install of GUFI-0.5.1-1.x86_64 conflicts with file from package filesystem-3.2-25.el7.x86_64

```

File: CMakeLists.txt [hunk 1]
```diff
@@ -297,14 +297,16 @@ set(CPACK_PACKAGE_VERSION_MAJOR ${GUFI_VERSION_MAJOR})
...
-execute_process(COMMAND uname -m OUTPUT_VARIABLE ARCH OUTPUT_STRIP_TRAILING_WHITESPACE)
-
 # Generate RPMs
 find_program(RPMBUILD rpmbuild)
 if (RPMBUILD)
   message(STATUS "Found rpmbuild: ${RPMBUILD_EXECUTABLE}")
+
+  # get the architecture of this machine
+  # assumes that we are not cross compiling
+  find_program(UNAME REQUIRED uname)
```
Comment (dmanno) [inline CMakeLists.txt]: Doesn't seem to be setting uname_executable correctly. Need some checking if(UNAME....). Currently creating RPM w/o arch set.

Comment (calccrypto) [reply CMakeLists.txt]: Apparently `find_program` does not have a `REQUIRED` argument. Fixed in 5b0a1034b31416d2b8fc0674a1641d8792668ee3.

**Comment by dmanno** [comment]
fine to merge.

**Pull request merged**